client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -890,6 +894,38 @@ public <T> CompletableFuture<T> executeRequestAsync(RequestContext context)
         return executor.executeRequestAsync(context);
     }
 
+    /**
+     * Gets list of files available at a Cassandra Instance using Sidecar during Live Migration.
+     *
+     * @param instance Sidecar instance where the request will be executed
+     * @return a CompletableFuture representing the completion of the operation
+     */
+    public CompletableFuture<InstanceFilesListResponse> liveMigrationListInstanceFilesAsync(SidecarInstance instance)
+    {
+        return executor.executeRequestAsync(requestBuilder().singleInstanceSelectionPolicy(instance)
+                                                            .request(new LiveMigrationListInstanceFilesRequest())
+                                                            .build());
+    }
+
+
+    /**
+     * Downloads requested file {@code reqPath} to the destination file provided with param {@code targetFile}
+     * during Live Migration.
+     *
+     * @param instance       the instance where the request will be executed
+     * @param reqPath        instance file path to copy
+     * @param targetFilePath destination file path to save file coming from remote
+     * @return a CompletableFuture representing the completion of the operation
+     */
+    public CompletableFuture<Boolean> liveMigrationStreamFileAsync(SidecarInstance instance, String reqPath, String targetFilePath)
```

Comment:
just curious, the boolean here can only be `true`, making it effectively the same as `CompletableFuture<Void>`, right? Then, you can remove the transformation at line926

Reply:
done